panacea/grpc_client.go

```diff
@@ -84,3 +133,109 @@ func (c *GrpcClient) GetDeal(id string) (markettypes.Deal, error) {
 
 	return *response.GetDeal(), nil
 }
+
+// RegisterDataValidator registers data validator on blockchain
+func (c *GrpcClient) RegisterDataValidator(address, endpoint string) error {
+	_, cancel := context.WithTimeout(context.Background(), time.Second)
+	defer cancel()
+	interfaceRegistry := makeInterfaceRegistry()
+	marshaler := codec.NewProtoCodec(interfaceRegistry)
+	txConfig := tx.NewTxConfig(marshaler, []signing.SignMode{signing.SignMode_SIGN_MODE_DIRECT})
+	txBuilder := txConfig.NewTxBuilder()
+
+	dataValidator := &datapooltypes.DataValidator{
+		Address:  address,
+		Endpoint: endpoint,
+	}
+
+	msgRegisterDataValidator := datapooltypes.NewMsgRegisterDataValidator(dataValidator)
+
+	err := txBuilder.SetMsgs(msgRegisterDataValidator)
+	if err != nil {
+		return err
+	}
+
+	conf := config.MustLoad()
```

Comment:
It's recommended to load the config once when the process is started at https://github.com/medibloc/panacea-data-market-validator/blob/2242b453db47738cceef6484553514646ad85c01/cmd/datavald/main.go#L12.
Based on the `config`, the [Service](https://github.com/medibloc/panacea-data-market-validator/blob/2242b453db47738cceef6484553514646ad85c01/server/service/service.go#L16) struct already contain the `ValidaterAccount` variable. 

Thus, I would add a `validatorAccount` argument to this `RegisterDataValidator()` instead of loading the entire config once again here.

Reply:
Okay, It's also call load the config twice. I'll add an argument `validatorAccount`.